test/integration/framework/constants.go

```diff
@@ -24,4 +24,12 @@ const (
 	Timeout            = time.Second * 10
 	ConsistentDuration = time.Second * 3
 	Interval           = time.Millisecond * 250
+
+	ResourcesMaxItems         = 16
+	FlavorsMaxItems           = 16
+	NodeSelectorMaxProperties = 8
+	TaintsMaxItems            = 8
+	PodSetsMaxItems           = 8
+	PodSetsMinItems           = 1
+	PodSetsCountMinItems      = 1
```

Comment:
just keep them in their respective tests packages. They serve no value as "global" variables.